capa/features/extractors/viv/function.py

```diff
@@ -80,7 +80,7 @@ def extract_function_loop(fhandle: FunctionHandle) -> Iterator[Tuple[Feature, Ad
                     bflags & envi.BR_COND
                     or bflags & envi.BR_FALL
                     or bflags & envi.BR_TABLE
-                    or bb.instructions[-1].mnem == "jmp"
+                    or bb.instructions[-1].mnem in ["jmp", "b", "bx"]
```

Comment:
```suggestion
                    or bb.instructions[-1].mnem in ("jmp", "b", "bx")
```

Reply:
I'm a little worried about mixing x86/amd64 and ARM throughout, so maybe we can add comments or separate them at least in some way for better maintainability?

Reply:
agree about mixing the logic, here and in other comments.

in general, i'd prefer to break out functions with human-readable names, like `is_jump` and `is_jump_x86`/`is_jump_arm` but i also recognize the overhead of making all these changes. please let us know what you're comfortable doing and what is reasonable for us to contribute, too.